documentation/documentation.gradle

```diff
@@ -77,7 +77,7 @@ asciidoctor {
 				'mainDir': project.sourceSets.main.java.srcDirs[0],
 				'testDir': project.sourceSets.test.java.srcDirs[0],
 				'testResourcesDir': project.sourceSets.test.resources.srcDirs[0],
-				'source-highlighter': 'coderay', // TODO switch to 'rouge' once supported by the html5 backend
+				'source-highlighter': 'rouge',
```

Comment:
The trick here is to set this as:

```
'source-highlighter': 'coderay@'
```

Then in the document header of index.adoc, add an override for PDF:

```
ifdef::backend-pdf[:source-highlighter: rouge]
```

(You could also flip the logic).